### DISCUSSION TITLE
Lyra

### DISCUSSION THREAD

**Discussion by kryokil** [Adriwin06/Ultimate-CommonUI-Menu-System#11] [Ideas]
Hey buddy, been watching your project for while. I did try to implement DLss in Lyra in the past but got sidetracked with project so then i abandoned it. How complicated would it be to adjust it for Lyra. I did quite bit of work mainly on combining Lyra and CitySample, my project actually using Lyra and some City sample. I will look on your latest release but I have feeling without touchin Lyra source it wont be easy to implement. Thank you for your hard work

**Comment by Adriwin06** [comment]
H! Thanks for you interrest in this project. I would be happy to help you but I don't understand whether you’re looking to adjust the entire project for Lyra or if you’re focusing solely on the DLSS implementation?

**Reply by kryokil** [reply]
at this point not sure what would be easier, to keep your menu or implement dlss into lyra menu, i did try this before but got stuck as most settings are in cpp and didnt achieve full functionallity.

are you working on some other projects too?

**Reply by Adriwin06** [reply]
If you want a complete menu system in your project that you understand, then I really recommend to use my project. If you want to add DLSS into Lyra, I think it would still be easier to add it in their menu system than adapting my system to Lyra, because everything is C++ driven and it seems complicated to change this if you are not experienced with C++. I personnaly don't have the knowledge to do that.
